platform/linux-generic/odp_packet_io.c

```diff
@@ -1,6 +1,6 @@
 /* SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Typos in commit comment: "alerady", "Implemeantation".

Reply:
v2